src/tools/pgindent/typedefs.list

```diff
@@ -1607,6 +1607,7 @@ ManyTestResourceKind
 Material
 MaterialPath
 MaterialState
+MdSMgrRelationData
```

Comment:
We also have `XLogSmgr`? And maybe nothing else defined in the "main" sources, but with this we have to be more careful.

Reply:
Oh, that is a good point. That makes the script kinda stupid as of my rewrite of it since it cares about which typedefs we used in `pg_tde.so` which `MdSMgrRelationData` is not while `XLogSmgr` is. Hmm.

Reply:
I have changed my approach in the patch to something sane, that I check everything except `pg_tde` with upstream's list and I check `pg_tde` with the combined list. Not sure how big improvement it is but it makes our diff against upstream smaller and is not stupid like my last approach.